app/templates/docs/enterprise.html

```diff
@@ -0,0 +1,625 @@
+<span id="index"></span>
+<h1 id="enterprise">enterprise<a class="headerlink" href="#enterprise" title="Permalink to this headline"></a></h1>
+<p>Enterprises are resources that bring together a set of organizations and members into a single collection. The enterprise resource makes heavy use of the SCIM API documented <a href="/advanced-reference/scim">here</a>. For a better understanding of the schema for enterprises, read the overview provided <a href="/advanced-reference/scim#scope">here</a>.</p>
+
+<div class="section" id="get-1-enterprises-identerprise">
+  <h2>GET <span class="target" id="enterprises-identerprise">/1/enterprises/[identerprise]</span><a class="headerlink" href="#get-1-enterprises-identerprise" title="Permalink to this headline"></a></h2>
```

Comment:
Elsewhere in the API docs, we use camelcase — i.e. `idEnterprise` instead of `identerprise`